apps/ejabberd/include/ejabberd.hrl

```diff
@@ -60,11 +55,11 @@
 -define(CRITICAL_MSG(Format, Args),
     lager:critical(Format, Args)).
 
--record(session, {sid      :: ejabberd_sm:sid(),
-                  usr      :: ejabberd:simple_jid(),
-                  us       :: ejabberd:simple_bare_jid(),
-                  priority :: integer(),
-                  info     :: list()
+-record(session, {sid,
```

Comment:
This record is used for selecting from Mnesia, so some fields can be set to `'_'` or `'$X'` then. Precise specs are moved here:
https://github.com/esl/MongooseIM/pull/508/files#diff-67c8b548efe99cc4b398cdbd4e7734e0R74